src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
I took a look at that bug and the referenced documentation and I think Livar's question is valid here. 

https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/specs/runtime-configuration-file.md#appnameruntimeconfigjson

The RuntimeConfigJson was always intended to be an output file and not something the user hand creates in their source directory. The doc specifies it should be user editable but that is in the scope of it already had been output by build.